addons/io_scene_gltf2/blender/exp/gltf2_blender_extract.py

```diff
@@ -676,8 +676,9 @@ def extract_primitives(glTF, blender_mesh, blender_vertex_groups, modifiers, exp
                     if modifiers is not None:
                         modifiers_dict = {m.type: m for m in modifiers}
                         if "ARMATURE" in modifiers_dict:
-                            armature = modifiers_dict["ARMATURE"].object
-                            skin = gltf2_blender_gather_skins.gather_skin(armature, export_settings)
+                            modifier = modifiers_dict["ARMATURE"]
+                            armature = modifier.object
+                            skin = gltf2_blender_gather_skins.gather_skin(armature, modifier.id_data, export_settings)
```

Comment:
It's possible for `modifier.object` to be `None` – here's an example: https://github.com/KhronosGroup/glTF-Blender-IO/issues/276#issuecomment-460751725. In that case, this will throw an error further down the call stack.

Reply:
Added a check, hope this is what you had in mind :)